go/Makefile

```diff
@@ -78,20 +78,21 @@ format:
 	golines -w --shorten-comments -m 127 .
 
 # unit tests - skip complete IT suite (including MT), and examples
+# due to a bug in go 1.20 and 1.21, we need to use -run flag to get -skip flag to work
 unit-test:
 	mkdir -p reports
 	set -o pipefail; \
 	LD_LIBRARY_PATH=$(shell find . -name libglide_rs.so|grep -w release|tail -1|xargs dirname|xargs readlink -f):${LD_LIBRARY_PATH} \
-	go test -v -race ./... -skip 'TestGlideTestSuite|Example*' $(if $(test-filter), -testify.m $(test-filter)) \
+	go test -v -race ./... -skip 'Example|TestGlideTestSuite' -run Test \
```

Comment:
Please restore `test-filter` there

Reply:
I can't at this point.  I need to get it so it will work in go 1.20 and I could not get the `-testify.m` flag to work there.  We can fix it in another PR, but can't keep delaying this PR without jeapardizing the 1.3 release date.

Reply:
Done.